lib/publiccloud/provider.pm

```diff
@@ -490,6 +491,7 @@ sub terraform_apply {
     }
 
     # Return an ARRAY of objects 'instance'
+    publiccloud::instances::set_instances(@instances);
```

Comment:
Just export the subroutines.

```suggestion
    set_instances(@instances);
```

Reply:
Maybe @cfconrad could be interested in this.

Reply:
Thx for pointing. Interesting point to use singleton here for test data!